test/test_config.rb

```diff
@@ -274,6 +274,14 @@ def test_config_files_with_float_convert
     assert_equal Float::INFINITY, conf.options[:max_fast_inline]
   end
 
+  def test_config_files_with_symbol_convert
+    conf = Puma::Configuration.new(config_files: ['test/config/with_symbol_convert.rb']) do
+    end
+    conf.load
+
+    assert_equal :ruby, conf.options[:io_selector_backend]
+  end
```

Comment:
Should we add a test that starts Puma with a different backend than the default (auto)?

Reply:
Sure, let me add a test for that, thanks for the suggestion! I, unfortunately, can't test for a specific backend since the available ones will vary from platform to platform. Unless I enforce `NIO4R_PURE=true` which I'm not very fond of.